omnilib/prototypes/recipe-generation.lua

```diff
@@ -161,6 +161,8 @@ setmetatable(BuildChain, {
 	end
 })
 
+local find_result_icon = omni.lib.find_result_icon
+
```

Comment:
Doesnt really save much, i would use the whole function name to avoid confusion

Reply:
I get scope issues if I call it directly.
``__omnimatter__/data.lua:16: __omnilib__/prototypes/recipe-generation.lua:742: attempt to index field 'omni' (a nil value)
``

Reply:
Resolved.